### ISSUE TITLE
android.hardware.fingerprint.FingerprintManager is Deprecated in API 28

### ISSUE THREAD

**Issue by skauss** [Mauin/RxFingerprint#103]
Hi,
have you seen the android.hardware.fingerprint.FingerprintManager is Deprecated in API 28
Regards
Stephan

**Comment by Mauin** [comment]
Yes, I'm aware the `FingerprintManager` is marked as `Deprecated` in API 28 and will be replaced by `BiometricDialog`. 
With the release of the first Android P developer preview I've started a `3.x` branch which tries to wrap both `FingerprintManager` and `BiometricDialog` depending on the device SDK. However due to time constraints I wasn't able to fully finish that work yet.
